tools/cli/helpers/json.js

```diff
@@ -17,8 +17,13 @@ function readJson( project, packageManager, output ) {
 	try {
 		data = fs.readFileSync( `projects/${ project }/${ file }`, 'utf8' );
 	} catch {
-		output && log( chalk.yellow( `This project does not have a ${ file } file.` ) );
-		return undefined;
+		output &&
+			log(
+				chalk.yellow(
+					`The '${ project }' project does not have a ${ file } file. Maybe there was a tpyo?`
+				)
+			);
+		process.exit( 1 );
```

Comment:
I'm wary about blindly exiting here, that seems like a recipe for unexpected behavior that's hard to deal with.

If we don't like returning undefined for the caller to handle, I'd rather we throw so the caller at least still has the option to catch it.

Reply:
👍 I've gone ahead and moved the exit later.